app/frontend/services/api/index.ts

```diff
@@ -224,10 +224,12 @@ export class Api {
     })
   }
 
+  async generatePermitApplicationMissingPdfs(id: string) {
+    return this.client.post<never>(`/permit_applications/${id}/generate_missing_pdfs`)
+  }
+
   async submitPermitApplication(id, params) {
```

Comment:
remove params if no longer needed?

Reply:
was mistakenly removed from `submitPermitApplication` so putting it back